src/main/java/org/broadinstitute/consent/http/service/EmailService.java

```diff
@@ -244,7 +245,7 @@ public void sendResearcherDarApproved(
       Integer researcherId,
       List<DatasetMailDTO> datasets,
       String dataUseRestriction)
-      throws Exception {
+      throws TemplateException, IOException {
```

Comment:
This fixes a set of warnings from sonar, it would like us to use more specific exception types here.